README.md

```diff
@@ -78,6 +78,27 @@ Follow this guide to setup your development machine.
     ```
     python scripts/workers/submission_worker.py
     ```
+#### Using Docker
+
+You can also use Docker Compose to run all the components of EvalAI together. The steps are:
+
+1. Get the source code on to your machine via git.
+
+    ```shell
+    git clone https://github.com/Cloud-CV/EvalAI.git evalai
+    ```
+
+2. Build the Docker containers. This might take a while.
+
+    ```
+    docker-compose -f docker-compose.dev.yml build
+    ```
+    
+3. Finally, run your containers with `docker-compose` and you should be able to access EvalAI at `localhost:8888`.
+
+    ```
+    docker-compose -f docker-compose.dev.yml up
```

Comment:
Let's combine step 2 and 3. It can be single command `docker-compose -f docker-compose.dev.yml up -d --build` which will build the containers before running.